scripts/bootstrap.js

```diff
@@ -36,7 +36,8 @@ function authorize() {
         });
 }
 
-function getOrCreateCollection(collection, token) {
+function getOrCreateCollection(collection, token, attributes) {
+    attributes = attributes || {};
```

Comment:
Interesting... and from the ES6 compatibility table, it appears you're right. Sorry about that.
https://kangax.github.io/compat-table/es6/

That said, keep an eye out- insofar as Node support tracks Chrome's JS engine, the upcoming Chrome 49 release will probably be followed by a significant improvement in Nodejs ES6 capabilities.